test/harness/e2e/vm/domain-template.xml

```diff
@@ -14,10 +14,13 @@
   <on_poweroff>destroy</on_poweroff>
   <on_reboot>restart</on_reboot>
   <on_crash>destroy</on_crash>
-  <os>
+  <os firmware='efi'>
     <type machine='q35'>hvm</type>
     <boot dev='hd'/>
-
+    <firmware>
+      <feature enabled='yes' name='secure-boot'/>
```

Comment:
😎

Reply:
Hmmm not sure if this is making it fail to boot on github CI :-( checking

Reply:
4 days later... https://github.com/flightctl/flightctl/pull/231/files#diff-29fa4e9e9dfff8f099fe2fdc77e1e702165851b9a835991d8d4a1f7cd14a71beR17